ang/civicase/ActivityFeed.js

```diff
@@ -432,25 +425,29 @@
        */
       function affixActivityDetailsPanel () {
         $timeout(function () {
-          $activityDetailsPanel = $element.find('.panel');
+          $activityDetailsPanel = $element.find('.civicase__activity-panel');
           $filter = $('.civicase__activity-filter');
           $feedListContainer = $('.civicase__activity-feed__list-container');
           $tabs = $('.civicase__dashboard').length > 0 ? $('.civicase__dashboard__tab-container ul.nav') : $('.civicase__case-body_tab');
           $toolbarDrawer = $('#toolbar');
```

Comment:
what if the toolbar module is not used? Or if we're not even using drupal?

Reply:
If the toolbar is not used, the height will just be 0, so it would still work. But I understand this is not the best solution, can we leave this out for the next PR? (as I dont have time left today)